+php-laravel-mode/.yas-parents.el

```diff
@@ -0,0 +1,26 @@
+(eval-when-compile (require 'subr-x))
+
+(defun string-split-words (string)
```

Comment:
Please rename to `+php-laravel-mode--string-split-words` (or similar). Wordy, I know, but `string-split-words` stands too high a chance of conflicting with future APIs.

Reply:
Yeah you're right, it's only used in 1 place so I might just inline it